nifi-nar-bundles/nifi-extension-utils/nifi-processor-utils/src/main/java/org/apache/nifi/processor/util/pattern/Put.java

```diff
@@ -89,7 +89,7 @@ public void onTrigger(ProcessContext context, ProcessSession session, FC functio
                 session.transfer(routedFlowFiles, relationship)));
 
         if (flowFiles == null || flowFiles.isEmpty()) {
-            logger.debug("No incoming FlowFiles.");
+            //logger.debug("No incoming FlowFiles.");
```

Comment:
This should probably be restored as it appears to be useful for debugging live flows